decidim-core/app/presenters/decidim/resource_presenter.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A presenter to render attributes for resources
+  class ResourcePresenter
+    include Decidim::TranslatableAttributes
+    include Decidim::SanitizeHelper
+
+    def initialize(organization)
```

Comment:
⚠️ I find this class a bit confusing. Based on the name and the location, it should follow the presenter pattern, receiving the resource to be presented. But it receives the organization and it's used inside other presenters.

So, what do you think about moving this class into the `decidim-core/lib/decidim` folder and converting it into a concern (you can check other concerns on that folder, as [Coauthorable](https://github.com/decidim/decidim/blob/develop/decidim-core/lib/decidim/coauthorable.rb))? After that, you could include that concern on all the affected presenters, without having to create a new instance to call `title` or `handle_locales`. Finally, if you need to access to the organization of the resource, maybe you can delegate that call to `__getobj__`, as all of the affected presenters inherit from `SimpleDelegator`.

Reply:
This cannot be done as a module like "Coauthorable". We'll use the Inheritance for this. Pending PR.